Project.toml

```diff
@@ -4,6 +4,9 @@ version = "0.1.0"
 
 [deps]
 FFTW = "7a1cc6ca-52ef-59f5-83cd-3a7055c09341"
+Images = "916415d5-f1e6-5110-898d-aaa5f9f070e0"
```

Comment:
`Images` and `Statistics` should be put in `compat` section

Reply:
I'm not too familiar with the `compat` section but I did just read [Pkg compatibility](https://julialang.github.io/Pkg.jl/v1/compatibility/#). What version of Statistics should I put? and should I also add FFTW and Images, too? Why not?

Reply:
There's some useful info in https://discourse.julialang.org/t/announcement-automatic-merging-for-the-general-registry/29961. In short, this issue will crop up only when you go to register the package. As far as which versions to put in, since this is a new package it would be fine to only include the versions you are using (e.g., `pkg> st ImageReconstruction`) since you know those work. Then the compat bounds can grow in a forward direction.